src/main/java/uk/gov/pay/ledger/util/csv/FlatCsvTransaction.java

```diff
@@ -0,0 +1,245 @@
+    private String email;
+
+    @JsonProperty("Amount")
+    private String amount;
+
+    @JsonProperty("Card Brand")
+    private String cardBrand;
+
+    @JsonProperty("Cardholder Name")
+    private String cardholderName;
+
+    @JsonProperty("Card Expiry Date")
+    private String cardExpiryDate;
+
+    @JsonProperty("Card Number")
+    private String cardNumber;
+
+    @JsonProperty("State")
+    private String state;
+
+    @JsonProperty("Finished")
+    private Boolean finished;
+
+    @JsonProperty("Error Code")
+    private String errorCode;
+
+    @JsonProperty("Error Message")
+    private String errorMessage;
+
+    @JsonProperty("Provider ID")
+    private String providerId;
+
+    @JsonProperty("GOV.UK Payment ID")
+    private String transactionId;
+
+    @JsonProperty("Issued By")
+    private String issuedBy;
+
+    @JsonProperty("Date Created")
+    private String dateCreated;
+
+    @JsonProperty("Time Created")
+    private String timeCreated;
+
+    @JsonProperty("Corporate Card Surcharge")
+    private String corporateSurcharge;
+
+    @JsonProperty("Total Amount")
+    private String totalAmount;
+
+    @JsonProperty("Wallet Type")
+    private String walletType;
+
+    @JsonProperty("Card Type")
+    private String cardType;
+
+    public FlatCsvTransaction() {
+
+    }
+
+    public FlatCsvTransaction(
+            String reference, String description, String email, String amount, String cardBrand, String cardholderName,
+            String cardExpiryDate, String cardNumber, String state, Boolean finished, String errorCode,
+            String errorMessage, String providerId, String transactionId, String issuedBy, String dateCreated,
+            String timeCreated, String corporateSurcharge, String totalAmount, String walletType, String cardType
+    ) {
+        this.reference = reference;
+        this.description = description;
+        this.email = email;
+        this.amount = amount;
+        this.cardBrand = cardBrand;
+        this.cardholderName = cardholderName;
+        this.cardExpiryDate = cardExpiryDate;
+        this.cardNumber = cardNumber;
+        this.state = state;
+        this.finished = finished;
+        this.errorCode = errorCode;
+        this.errorMessage = errorMessage;
+        this.providerId = providerId;
+        this.transactionId = transactionId;
+        this.issuedBy = issuedBy;
+        this.dateCreated = dateCreated;
+        this.timeCreated = timeCreated;
+        this.corporateSurcharge = corporateSurcharge;
+        this.totalAmount = totalAmount;
+        this.walletType = walletType;
+        this.cardType = cardType;
+    }
+
+    private static String penceToCurrency(Long amount) {
+        if (amount != null) {
+            DecimalFormat decimalFormat = new DecimalFormat("#,##0.00");
+            return decimalFormat.format(amount / 100);
+        }
+        return null;
+    }
+
+    public static FlatCsvTransaction from(TransactionView transactionView) {
```

Comment:
Could use a test/few to make sure it works as intended and there won't be any regression in the future

Reply:
Added a test for the content too, agreed we should add a lot more once we've decided how generating should work.